### PR TITLE
Update caller doc

### PR DESCRIPTION
**Pull request by harrislapiroff**
* Add wireless mic info
* Clarify feedback process

### CODE REVIEW

**Comment by harrislapiroff** [comment]
I did check with the sound list about updating the doc to include wireless mic info and no one responded, which I took as implicit approval.

**Review by jeffkaufman — APPROVED** [review]

File: caller-welcome.html [hunk 1]
```diff
@@ -39,7 +39,9 @@ <h1>Caller Welcome</h1>
   <li><p>Sound:
       <ul>
         <li>We'll provide sound.
-        <li>Our caller's mic is a wired mic on a stand.
+        <li>We have both wireless and wired mics. We typically will set
```
[the remark this answers is not in the document]
Comment (harrislapiroff) [reply caller-welcome.html:42]: I think we actually also have a lavalier if they prefer! Michael's kit came with both

[the remark this answers is not in the document]
Comment (jeffkaufman) [reply caller-welcome.html:42]: Can we say that then? ;)

[the remark this answers is not in the document]
Comment (harrislapiroff) [reply caller-welcome.html]: Done! Merging!

**Pull request merged**